assignment/api/src/resources/assignment.properties

```diff
@@ -178,6 +178,7 @@ gen.orisub2       = Original submission text with the instructor's comments inse
 gen.orisub3       = Last submission text with instructor comments inserted if applicable
 gen.outof         = out of
 gen.peerReviews   = Peer Reviews
+gen.pending_resubmit = Returned - Pending Resubmission
```

Comment:
"Pending" doesn't seem like the correct word to use here. Sounds like student is in progress or something. Although, I don't completely understand the use case you're trying to address here. Can you provide some context about this fix?

Would "Returned - Resubmission Available" be better?